### PR TITLE
GPII-3546: Redesign of the QuickStrip

### PR DESCRIPTION
**Pull request by georgitodorov**

### CODE REVIEW

File: src/main/dialogs/quickSetStrip/qssTooltipDialog.js [hunk 1]
```diff
@@ -39,9 +39,11 @@ fluid.defaults("gpii.app.qssTooltipDialog", {
         "gpii.app.dialog.offScreenHidable"
     ],
 
+    arrowWidth: 18,
```
Comment (danailbd) [inline src/main/dialogs/quickSetStrip/qssTooltipDialog.js:42]: How about grouping such "renderer related" hardcoded sizes under a property like `dialogElementsMetrics`

File: src/main/dialogs/quickSetStrip/qssWidgetDialog.js [hunk 2]
```diff
@@ -51,7 +59,7 @@ fluid.defaults("gpii.app.qssWidget", {
                 funcName: "gpii.app.scale",
                 args: [
                     "{that}.options.scaleFactor",
-                    637
+                    400
```
Comment (danailbd) [inline src/main/dialogs/quickSetStrip/qssWidgetDialog.js:62]: Shouldn't we have this as an option value?

File: src/main/dialogs/quickSetStrip/qssWidgetDialog.js [hunk 3]
```diff
@@ -60,7 +68,7 @@ fluid.defaults("gpii.app.qssWidget", {
                 funcName: "gpii.app.scale",
                 args: [
                     "{that}.options.scaleFactor",
-                    627
+                    365
```
Comment (danailbd) [inline src/main/dialogs/quickSetStrip/qssWidgetDialog.js:71]: See previous

File: src/main/dialogs/quickSetStrip/qssWidgetDialog.js [hunk 4]
```diff
@@ -207,9 +215,12 @@ gpii.app.qssWidget.toggle = function (that, setting, btnCenterOffset, activation
  * the screen.
  */
 gpii.app.qssWidget.getWidgetPosition = function (that, btnCenterOffset) {
+    var extraVerticalOffset = that.options.extraVerticalOffset,
+        scaleFactor = that.options.scaleFactor;
+
     return {
         x: btnCenterOffset.x - that.width / 2,
-        y: btnCenterOffset.y
+        y: btnCenterOffset.y + scaleFactor * extraVerticalOffset
```
Comment (danailbd) [inline src/main/dialogs/quickSetStrip/qssWidgetDialog.js:223]: why do we also need to apply the `scaleFactor` to the `extraVerticalOffset`

File: src/main/dialogs/quickSetStrip/qssWidgetDialog.js (hunk 2 above)
Comment (georgitodorov) [reply src/main/dialogs/quickSetStrip/qssWidgetDialog.js:62]: This is changed in the branch for scaling up/down the QSS.

File: src/main/dialogs/quickSetStrip/qssTooltipDialog.js (hunk 1 above)
Comment (georgitodorov) [reply src/main/dialogs/quickSetStrip/qssTooltipDialog.js:42]: There is just one such property here so I think this is not needed.

File: src/main/dialogs/quickSetStrip/qssWidgetDialog.js (hunk 3 above)
Comment (georgitodorov) [reply src/main/dialogs/quickSetStrip/qssWidgetDialog.js:71]: See my previous answer :)

File: src/main/dialogs/quickSetStrip/qssWidgetDialog.js (hunk 4 above)
Comment (georgitodorov) [reply src/main/dialogs/quickSetStrip/qssWidgetDialog.js:223]: In order to preserve the proportion of the space between the QSS widget and the stip itself.

**Comment by georgitodorov** [comment]
Closing this PR as its code is already part of `morphic-1.1` and thus should not be merged in `morphic-1.0.1`.

**Pull request closed without merging**